CHANGES.md

```diff
@@ -14,6 +14,7 @@
 
 ##### Fixes :wrench:
 
+- Fixed diffuse color calculation for PBR materials. Many models will now appear slightly brighter. This also fixes an [issue affecting the KHR_materials_specular extension](https://github.com/CesiumGS/cesium/issues/12041). [#12043](https://github.com/CesiumGS/cesium/pull/12043)
```

Comment:
Even though they were fixed in one PR, I would suggest listing this as two different items. The former (`Fixed diffuse color calculation for PBR materials`) can link to the PR. The later (`fixes an issue affecting the KHR_materials_specular extension`) can link to the issue.